ci/licenses_golden/licenses_flutter

```diff
@@ -42733,8 +42733,6 @@ ORIGIN: ../../../flutter/fml/platform/darwin/scoped_block.h + ../../../flutter/L
 ORIGIN: ../../../flutter/fml/platform/darwin/scoped_block.mm + ../../../flutter/LICENSE
 ORIGIN: ../../../flutter/fml/platform/darwin/scoped_nsautorelease_pool.cc + ../../../flutter/LICENSE
 ORIGIN: ../../../flutter/fml/platform/darwin/scoped_nsautorelease_pool.h + ../../../flutter/LICENSE
```

Comment:
I was going to say surely this isn't used anywhere now since it wouldn't be ARC-friendly, and so we could clean this up too, and then I looked at the implementation. Oh my...

Reply:
Yeah I was looking at it this morning two. It's used in two or three places. I suspect most can actually be updated to `.mm` files and done properly, but the implementation as-is is also well... consistent with avoiding Obj-C at least, I guess.